docs/website/docs/reference/bindings/python.md

```diff
@@ -15,15 +15,22 @@ components:
 
 | PIP package name             | Description                                                                 |
 |------------------------------|-----------------------------------------------------------------------------|
-| `iree-compiler`     | IREE's generic compiler tools and helpers                                   |
-| `iree-runtime`      | IREE's runtime, including CPU and GPU backends                              |
-| `iree-tools-tf`     | Tools for importing from [TensorFlow](https://www.tensorflow.org/)          |
-| `iree-tools-tflite` | Tools for importing from [TensorFlow Lite](https://www.tensorflow.org/lite) |
-| `iree-jax`          | Tools for importing from [JAX](https://github.com/google/jax)               |
+| `iree-base-compiler` | IREE's generic compiler tools and helpers                                   |
+| `iree-base-runtime`  | IREE's runtime, including CPU and GPU backends                              |
+| `iree-tools-tf`      | Tools for importing from [TensorFlow](https://www.tensorflow.org/)          |
+| `iree-tools-tflite`  | Tools for importing from [TensorFlow Lite](https://www.tensorflow.org/lite) |
+| `iree-jax`           | Tools for importing from [JAX](https://github.com/google/jax)               |
 
 Collectively, these packages allow for importing from frontends, compiling
 towards various targets, and executing compiled code on IREE's backends.
 
+???+ Note "Note - `iree-compiler` and `iree-runtime` are deprecated"
+    The Python packages `iree-compiler` and `iree-runtime` have been
+    renamed to `iree-base-compiler` and `iree-base-runtime`
+    respectively, thus effectively deprecating the old packages.
+    This name change only affects the names of the packages but not
+    the modules.
```

Comment:
Depending on how pip behaves, we may want a tip to `pip uninstall iree-compiler iree-runtime` during the transition period (I just start with fresh venvs :P)

Reply:
I think it's fine on the Python bindings page, not the snippet that gets included everywhere. If we get user reports showing symptoms of multiple conflicting packages being installed then we can put it in the snippet too.

Reply:
Added a note. Let me know if you think this is ready to land.